app/scripts/api.js

```diff
@@ -664,7 +668,7 @@ const createApi = function api(context, pubSub) {
        * Set or get an option.
        * @param   {string}  key  The name of the option you want get or set
        * @param   {*}  value  If not `undefined`, `key` will be set to `value`
-       * @return  {[type]}  When `value` is `undefined` the current value of
+       * @return  {object}  When `value` is `undefined` the current value of
```

Comment:
Just curious: Is there any difference between `object` and  `obj`?

Reply:
No, and that notation has no functional significance. It's just to inform the user. With that said, consistency is nice so I'll update it to match the other return value.

```suggestion
       * @return  {obj}  When `value` is `undefined` the current value of
```